contrib/docker/DOCKER.md

```diff
@@ -0,0 +1,99 @@
+# Using Boolector with Docker
+
+Boolector has support for executing Boolector "input files" (e.g., `.smt2`)
+files via the use of a Docker container. The base container used is Alpine
+Linux.
+
+This guide presumes you are familiar with both Docker and Boolector, and are
+able to install the necessary prerequisites to run Docker.
+
+
+## Building the Docker image
+
+To build the Docker image locally, you can perform the following:
+
+```
+git clone https://github.com/Boolector/boolector.git
+cd boolector/contrib/docker
+docker build -t btor/btor .
+```
+
+This will then build you a fresh Docker image, with Boolector built inside.
+
+
+## Using Docker to run `.smt2` files
+
+As Boolector is able to read SMT2LIB files via `stdin`, this means that it is
+extremely easy to run Boolector against a `.smt2` file. For example:
+
+```
+#
+# Presuming you're at the root of a Boolector clone
+#
```

Comment:
Maybe remove the empty comment lines (to make it more compact)?

Reply:
Done!